bitnami/spark/README.md

```diff
@@ -82,6 +82,7 @@ The following tables lists the configurable parameters of the spark chart and th
 | `master.readinessProbe.timeoutSeconds`      | When the probe times out                                                                           | 5                                                       |
 | `master.readinessProbe.failureThreshold`    | Minimum consecutive failures for the probe to be considered failed after having succeeded.         | 6                                                       |
 | `master.readinessProbe.successThreshold`    | Minimum consecutive successes for the probe to be considered successful after having failed        | 1                                                       |
+| `master.podAnnotations` | Additional annotations for master metrics exporter                                                          | `{prometheus.io/scrape: "true", prometheus.io/port: "8080", prometheus.io/path: "/metrics/applications/prometheus}` |
```

Comment:
I would make the description just "Additional annotations for the master" since this is not exclusive for metrics. The default value is also `{}`.

Reply:
fixed